docs/src/components/sidebarIndex.js

```diff
@@ -30,59 +68,37 @@ const sidebarIndex: sidebarIndexType = [
     subsections: [
       {
         subsectionName: 'Data Display',
-        pages: ['Avatar', 'AvatarPair', 'Badge', 'GroupAvatar', 'Table'],
+        pages: componentSubSectionPages.displayDate,
       },
       {
         subsectionName: 'Feedback',
-        pages: ['Modal', 'Pulsar', 'Spinner', 'Toast'],
+        pages: componentSubSectionPages.feedBack,
       },
       {
         subsectionName: 'Foundation',
-        pages: ['Heading', 'Icon', 'Text'],
+        pages: componentSubSectionPages.foundations,
       },
       {
         subsectionName: 'Forms',
-        pages: [
-          'Button',
-          'Checkbox',
-          'DatePicker',
-          'Flyout',
-          'IconButton',
-          'Label',
-          'Pog',
-          'RadioButton',
-          'SearchField',
-          'SelectList',
-          'Switch',
-          'TapArea',
-          'TextArea',
-          'TextField',
-          'Tooltip',
-        ],
+        pages: componentSubSectionPages.forms,
       },
       {
         subsectionName: 'Layout',
-        pages: [
-          'Box',
-          'Card',
-          'Collage',
-          'Column',
-          'Container',
-          'Divider',
-          'Layer',
-          'Masonry',
-          'Row',
-          'Stack',
-          'Sticky',
-        ],
+        pages: componentSubSectionPages.layout,
       },
       {
         subsectionName: 'Media',
-        pages: ['Image', 'Letterbox', 'Mask', 'Video'],
+        pages: componentSubSectionPages.media,
       },
       {
         subsectionName: 'Navigation',
-        pages: ['Link', 'SegmentedControl', 'Tabs'],
+        pages: componentSubSectionPages.navigation,
+      },
+      {
```

Comment:
@mminute I like the fact that we now see all the components by default but it's odd to see them twice. For now, I'm going to remove the `All` section - we can always add it back in later.

![image](https://user-images.githubusercontent.com/127199/86388951-797c1c00-bc4a-11ea-9f95-2a351c16d4d6.png)

Reply:
This also breaks when you go `All => Avatar` & then refresh the page. Now `Data Display => Avatar` is selected.

Reply:
Ah, i see.  I didn't test the refresh case.

I get what you are saying about seeing the components twice, but as someone who is familiar with gestalt I found it slower to identify the component i wanted to see in the divided list rather than an alphabetical list.  The `All` option was meant to be a bailout for those who, like me, might still want a list